pkg/initiator/initiator.go

```diff
@@ -35,53 +42,74 @@ const (
 	waitDeviceInterval   = 1 * time.Second
 )
 
+var (
+	idGenerator         IDGenerator
+	isUblkTargetCreated = false
+)
+
 type Initiator struct {
-	Name               string
-	SubsystemNQN       string
-	UUID               string
-	TransportAddress   string
-	TransportServiceID string
+	Name     string
+	Endpoint string
+	dev      *util.LonghornBlockDevice
+	isUp     bool
 
-	Endpoint       string
-	ControllerName string
-	NamespaceName  string
-	dev            *util.LonghornBlockDevice
-	isUp           bool
+	NvmeTcpInfo *NvmeTcpInfo
+	UblkInfo    *UblkInfo
 
 	hostProc string
 	executor *commonns.Executor
 
 	logger logrus.FieldLogger
 }
 
-// NewInitiator creates a new NVMe-oF initiator
-func NewInitiator(name, subsystemNQN, hostProc string) (*Initiator, error) {
+type NvmeTcpInfo struct {
```

Comment:
How about `NVMeTCPInfo` for naming alignment?
```suggestion
type NVMeTCPInfo struct {
```

Reply:
Surely. Updated. Thank you